Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_armblade.yml

```
@@ -0,0 +1,50 @@
+        Structural: 20
+    clickPartDamageMultiplier: 2
+    heavyPartDamageMultiplier: 1.5
+    soundHit:
+      path: /Audio/Weapons/bladeslice.ogg
+  - type: Item
+    size: Ginormous
+    sprite: _Goobstation/Changeling/arm_blade.rsi
+  - type: Prying
+    speedModifier: 2
+    pryPowered: true
+  - type: UseDelay # For insta prying
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)